pages/wizard/subscriptions/add_subscriptions.py

```diff
@@ -0,0 +1,30 @@
+from selenium.webdriver.common.by import By
+from pages.base import Base
+
+
+class AddSubscriptions(Base):
+
+    # locators
+    _subs_row_loc = (By.XPATH,
+                     '//input[@name="isSelectedSubscription"]/../..')
+    _checkbox_loc = (By.XPATH, './/input[@name="isSelectedSubscription"]')
+    _quantity_loc = (By.XPATH, './/input[@name="qtyToAttach"]')
```

Comment:
It's probably fine, but when I ran this through my XPATH locator on my browser it said no matches found.